src/Tests/SmsFrameworkWebTest.php

```diff
@@ -15,6 +18,54 @@
 class SmsFrameworkWebTest extends SmsFrameworkWebTestBase {
```

Comment:
Lets not throw a bunch of random stuff into this class. Please create a dedicated reports class